types/index.d.ts

```diff
@@ -66,11 +66,31 @@ declare namespace StyleDictionary {
   }
 
   interface Prop {
-    original: Property;
+    /** A default name of the property that is set to the key of the property. */
     name: string;
-    attributes: Attributes;
-    path: string[];
     value: string;
+    /** The object path of the property.
+     *
+     * `color: { background: { primary: { value: "#fff" } } }` will have a path of `['color', 'background', 'primary']`.
+     */
+    path: string[];
+    /**
+     * A pristine copy of the original property object.
+     *
+     * This is to make sure transforms and formats always have the unmodified version of the original property.
+     */
+    original: Property;
+    /**
+     * The file path of the file the token is defined in.
+     *
+     * This file path is derived from the source or include file path arrays defined in the configuration.
+     */
+    filePath: string;
+    /**
+     * If the token is from a file defined in the source array as opposed to include in the [configuration](https://amzn.github.io/style-dictionary/#/config).
```

Comment:
In case the reviewer is unfamiliar with how these comments are presented, markdown links show as a link (which is clickable).

![image](https://user-images.githubusercontent.com/848525/102973467-b8bf6180-4537-11eb-98d0-3910297ef2c4.png)